MozillaFoundation/foundation.mozilla.org
# Feature/tp1 211 change cms snippets order

# Description

Change order of the snippet fields on CMS in order to have header and description field next to each other without newsletter field interfering.

# To test
- Pull branch locally
- Access the CMS, and got to Home > Snippets. You can open either Signups or Blog Signups. 
- Open any snippet listed and you will see the order of the fields as below image:

![image](https://github.com/user-attachments/assets/51fb679d-1a91-48d8-98d4-e80a2133a8f0)

![image](https://github.com/user-attachments/assets/bcff32b3-c757-4b2d-9651-d24158a7036e)


Link to sample test page: https://foundation-s-feature-tp-x6l9xv.herokuapp.com/
Related PRs/issues: https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11398#pullrequestreview-1743206684

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**
